app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -1,25 +1,32 @@
-<template lang="pug">
-  div
-    div(v-if="yourValidators.length > 0")
-      table-validators(:validators="yourValidators")
-
-    tm-data-msg(v-if="yourValidators.length < 1", icon="info_outline")
-      div(slot="title") No Active Delegations
-      div(slot="subtitle") Looks like you haven't delegated any {{ this.bondingDenom }}s yet. Head over to the #[router-link(:to="{name: 'Validators'}") validator list] to make your first delegation!
-
-    .check-out-message(v-if="yourValidators.length > 0")
-      | Check out
-      |
-      router-link(:to="{name: 'Validators'}") the validator list
-      |
-      | to find other validators to delegate to.
-
-    div(v-if="undelegatedValidators.length")
-      h3.tab-header
-        | Inactive Delegations
-        |
-        i.material-icons.info-button(v-tooltip.top="unbondInfo") info_outline
-      table-validators(:validators="undelegatedValidators")
+<template>
+  <div>
+    <div v-if="yourValidators.length &gt; 0">
+      <table-validators :validators="yourValidators" />
+    </div>
+    <tm-data-msg v-if="yourValidators.length &lt; 1" icon="info_outline">
```

Comment:
should we keep these `&lt;` and `&gt;`  instead of  `<` `>` ?

Reply:
no, probably the reason for some of the bugs

Reply:
fixed